pkg/tcpip/iptables/iptables.go

```diff
@@ -151,11 +151,20 @@ func (it *IPTables) Check(hook Hook, pkt tcpip.PacketBuffer) bool {
 		// If the table returns Accept, move on to the next table.
 		case Accept:
 			continue
+
 		// The Drop verdict is final.
 		case Drop:
 			return false
-		case Stolen, Queue, Repeat, None, Jump, Return, Continue:
+
+		// Return is invalid here. A built-in chain can return, in which case
+		// we should have underflowed into a default policy. Default policies
+		// can only be either ACCEPT or DROP.
+		case Return:
+			panic("Tables shouldn't be able to return Return.")
```

Comment:
What can hit these panics? Are you guaranteed that you wouldn't be able to generate this in practice?